packages/make-styles/src/ax.ts

```diff
@@ -0,0 +1,75 @@
+import { HASH_PREFIX_CODE, RTL_PREFIX_CODE } from './constants';
+
+/**
+ * This length includes a leading character "f" or "r", e.g. `"f1s4A"` would be a valid atomic property hash.
+ */
+const ATOMIC_PROPERTY_LENGTH = 5;
+
+/**
+ * Function can take any number of arguments, joins classes together and deduplicates atomic declarations.
+ *
+ * Atomic declarations take the form of `f{property}{value}` or f{property}{value}`, where both `property` and `value`
+ * are hashes **four characters long**.
+ *
+ * Classnames can be of any length, this function can take both atomic declarations and class names.
+ *
+ * Input:
+ * ```
+ * ax('ui-button', 'faaaabbbb', 'fccccdddd')
```

Comment:
This shouldn't this give the intended output ? 

```suggestion
 * ax('ui-button', 'fccccbbbb', 'fccccdddd')
```

Reply:
```suggestion
 * ax('ui-button', 'faaaabbbb', 'faaaadddd')
```

Good catch 👍